docs/portal/registry-users/deleting-users.md

```diff
@@ -0,0 +1,5 @@
+# Deleting Users
```

Comment:
This operation requires admin portal user permission. Please add a caveat on this page.

Reply:
Done.